webshare/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        setupManualShareTest(false, '', '', location.toString());
```

Comment:
Aha, didn't know about baseURI. Updated, and also updated `getAbsoluteUrl` in the helper file.

I've changed share-url-relative-manual.html to test with a custom `<base>` to ensure the implementation uses baseURI and not location. Chrome does this correctly.

[Related spec bug](https://github.com/WICG/web-share/issues/36).